Project-Code/src/index.js

```diff
@@ -59,26 +58,36 @@ const user = {
 
 
 app.get('/', (req, res) =>{
-    
     res.render('pages/login')
 });
 
 app.get('/test', (req, res)=>{
+    //Making db query to retrieve address before API call to get voterInfo
+    const query = "SELECT addressLine1, addressLine2, city, state, zip_code FROM voters WHERE username='test';";
+    const values = [user];
+
+    db.any(query, values)
+        .then(async (data) =>{
+            address = data.addressLine1 + data.city + data.state + data.zip_code;
+      console.log(data);
+      console.log(address);
+      console.log(data.addressLine1);
+        })
+        .catch((error) => {
+                //hanlde errors
+                console.log(error);
+                res.redirect("/");
+        });
+    //API call
+    console.log(values);
     axios({
```

Comment:
Look at using `axios.create` to make an instance with the api as a base url. Something like

```js
const civicRequest = axios.create({
 baseURL: 'https://civicinfo.googleapis.com',
  params: { 'apiKey': API_Key } // this will append the params for you
  });
```

then you can use`civicRequest.get('civicinfo/v2/voterinfo', { params: { you can add more params here } });`